test/tools/scripts/extract-testcases.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env python3
```

Comment:
Weird that this does not have source highlighting...

Reply:
Maybe just place this under `/scripts`? We need to clean that up at some point, but better to keep everything in one place until then.

Reply:
oki